Проектирование ERD-диаграммы

AlexandraLis295 opened the pull request:
1. Спроектировала ERD-диаграмму ПрО "Компания по услугам связи"

AlexandraLis295 committed:
1. Спроектировала ERD-диаграмму ПрО "Компания по услугам связи"

MarinaPaley reviewed (changes requested):
1, Не поняла по первичным ключам (где-то есть, где-то нет)
2. Наименования (ID - для ПК, ID_ИмяГлавнойТаблицы, таблица, разрывающая связь многие-ко-многим - ИмяГлавной_ИмяПодчиненной, все остальное -  в CamelCase
3. Нет связи между таблицами Подписчик и Телефонные номера

AlexandraLis295 committed:
1. Исправила первичные ключи

2. Подправила наименования (свела к CamelCase)
3. Добавила отсутствующую ранее связь между таблицами
